website/slides/dashboard.md

```diff
@@ -0,0 +1,90 @@
+layout:true
+<div class="slide-footer">
+    <div class="slide-copright col">
+        <img class="image logo" width="70"/>
+        <span>WaveMaker.Inc. All Rights Reserved</span>
+    </div>
+    <div class="action-btns col">
+        <button title="Previous (←)" class="previousSlide"><div class="arrow-left"></div></button>
+        <button title="Next (→)" class="nextSlide"><div class="arrow-right"></div></button>
+        <button title="Full screen (F)" class="fullScreen"><i class="wms wms-fullscreen"></i></button>
+    </div>
+</div>
+
+---
+class:wave-theme,master
+
+### Dashboard
+Once you have created projects, the **Project Dashboard** page displays all your Projects, Prefabs and Template Bundles along with the latest activity updates.
+
+---
+layout:true
```

Comment:
Should this HTML code be repeated in every slideshow markdown file? Is there a way to reference it from one place, so there is no duplication?